compiler/natives/src/syscall/js/js.go

```diff
@@ -22,27 +22,23 @@ const (
 	TypeFunction
 )
 
+// Same order as Type constants
+var typeNames = []string{
+	"undefined",
+	"null",
+	"boolean",
+	"number",
+	"string",
+	"symbol",
+	"object",
+	"function",
+}
```

Comment:
Doing this as a slice because lookups *should* be faster than a switch statement I believe, plus it makes the `getValueType()` implementation easier to match the types.